### PR TITLE
close #266 for revalidate with onChange

### LINKED ISSUE #266: Errors doesn't clear on change while using `onBlur` mode
**Describe the bug**
I'm not sure if this is a bug. But while using `onBlur` mode, errors don't clear when a field is changed like they do while using `onSubmit` mode. 

**To Reproduce**
[Here](https://codesandbox.io/s/peaceful-framework-rphl3?fontsize=14) is a recreation.

**Expected behavior**
I expect the errors to be cleared when the corresponding field changes.

### PR DESCRIPTION
**Pull request by bluebill1049**

### CODE REVIEW

**Review by barrymay — DISMISSED** [review]
nothing glaring to me as an issue, just more style issues

File: src/useForm.ts [hunk 1]
```diff
@@ -454,41 +453,41 @@ export default function useForm<
...
+    const currentField = fields[name];
     const existRadioOptionIndex =
-      isRadio && field && isArray(field.options)
-        ? field.options.findIndex(({ ref }: Field) => value === ref.value)
+      isRadio && currentField && isArray(currentField.options)
+        ? currentField.options.findIndex(({ ref }: Field) => value === ref.value)
         : -1;
 
-    if ((!isRadio && field) || (isRadio && existRadioOptionIndex > -1)) return;
+    if (isRadio ? existRadioOptionIndex > -1 : currentField) return;
```
[the remark this answers is not in the document]
Comment (bluebill1049) [reply src/useForm.ts]: good pick up :) this is specific for the radio button.

if the element is found (not radio) which means register already done, then skip.

if the element is found and radio, check if radio option is registered if so skip.

I think what I had is accurate.

**Comment by bluebill1049** [comment]
thanks for the review @barrymay 🙏

File: src/useForm.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (barrymay) [reply src/useForm.ts]: yeah I'm just saying that you're already checking for isRadio in line 459 and falling to -1, so by then it's going to be a -1 for sure.. either way is fine, just again seeing there ternary in the if is weird, but what you had is ok

[the remark this answers is not in the document]
Comment (bluebill1049) [reply src/useForm.ts]: I will improve on it. Thanks Barry

**Pull request merged**